src/utils/miscellaneous.ts

```diff
@@ -91,6 +91,24 @@ export function getImageUrl(tokenAddress?: string): string | undefined {
   }
 }
 
+export function isNativeToken(address: string): boolean {
+  return address.toLowerCase() === NATIVE_TOKEN_ADDRESS.toLowerCase()
+}
+
+export function getImageAddress(address: string, network: Network): string {
+  if (isNativeToken(address)) {
+    // What is going on here?
+    // Well, this address here is the path on `src/assets/tokens/`
+    // So these special values will use the local images,
+    // because they are native tokens and don't really have an address
+    if (network === Network.xDAI) {
```

Comment:
nit: sorry i hate nested `if`s sometimes and think it would be great mmkay to see
```ts
if (isNativeToken(address)) {
    // What is going on here?
    // Well, this address here is the path on `src/assets/tokens/`
    // So these special values will use the local images,
    // because they are native tokens and don't really have an address
    return network === Network.xDAI ? 'xdai' : 'eth'
  }
// ... //
```